index.html

```diff
@@ -52,18 +52,30 @@ <h1 class="sr-only">Bikes for Refugees Scotland - Providing donated bikes and ac
 			</div>
 			<div class="row">
 				<div class="primary-content col-lg-8">
-					<div class="jumbotron">
+					<div class="jumbotron col-lg-12">
+						<div class="back-contents col-lg-12">
+							<img src="images/header-bike.jpg">
+						</div>
+						<div class="over-back-contents col-lg-12">
 						<h1 class="display-3">Bikes for Refugees</h1>
 						<p class="lead">Providing donated bikes and accessories to refugees and asylum seekers in Scotland.</p>
 						<hr class="my-4">
 						<div class="buttons">
 							<a class="btn btn-primary btn-lrg" href="#">Donate a bike today</a>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
+						</div>
+					</div>
+
+					<div class="three-boxes col-lg-12">
```

Comment:
You cannot add any custom horizontal padding, margin or width to the same level as a `col-` class as it will interfere with it.